app/controllers/comment_controller.php

```diff
@@ -0,0 +1,55 @@
+<?php
+class CommentController extends AppController
+{
+    public function edit_comment()
```

Comment:
function name may seem redundant, so I think it's better to name this as "edit" only instead :bow: (Apply to others. If the name of the class already states it, no need to include the name on the function name :dart: )